app/src/main/res/layout/fragment_wallpaper_settings.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <androidx.compose.ui.platform.ComposeView
+        android:id="@+id/compose_view"
```

Comment:
nit: I think the background of the `compose_view` should be the same as the wallpaper screen :)

Dark mode
<img width="458" alt="image" src="https://user-images.githubusercontent.com/773158/147150323-2593f699-b7cd-494b-8f51-2fe877991d6b.png">

Light mode
<img width="457" alt="image" src="https://user-images.githubusercontent.com/773158/147150518-a76cb207-b2a7-44fb-9ff3-dd4cddb6db8f.png">

Reply:
Good catch! The light mode one matches the design, which has the wallpapers on a surface rather than directly on the background. I believe this is so the default wallpaper can still be seen, since it's the same color. I'll update the PR such that dark mode reflects that behavior, though

Reply:
👍🏽